lib/handlers/shared.ts

```diff
@@ -93,8 +93,8 @@ export const DEFAULT_ROUTING_CONFIG_BY_CHAIN = (chainId: ChainId): AlphaRouterCo
         },
         maxSwapsPerPath: 3,
         minSplits: 1,
-        maxSplits: 7,
-        distributionPercent: 5,
+        maxSplits: 2,
+        distributionPercent: 25,
```

Comment:
this pr now affects all chains (mainnet too). I think we want to case it for zksync only right? cc: @jsy1218 @zhongeric 

I was wondering why our latency wnet down too.. but this explains it

Reply:
i was also checking the latency metrics today, and ended up to this PR as well.
Agree this needs to apply to zksync only

Reply:
fixed in https://github.com/Uniswap/routing-api/pull/746/files